### PR TITLE
added method getTicketsByStadiumSector

### PR DESCRIPTION
**Pull request by AnyaRatusnyak**

### CODE REVIEW

File: src/TicketService.java [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+
+        List<Ticket> ticketsFromStadiumSector = ticketService.getTicketsByStadiumSector(Ticket.StadiumSector.A);
+        if (!ticketsFromStadiumSector.isEmpty()) {
+            System.out.println("Found tickets:");
+            for (Ticket item : ticketsFromStadiumSector) {
+                System.out.println(item);
+            }
+        } else {
+            System.out.println("No tickets found for the given stadium sector.");
+        }
```
Comment (Alexmansar) [inline src/TicketService.java:23]: in general could be separate method

File: src/TicketService.java [hunk 2]
```diff
@@ -35,4 +48,14 @@ public Ticket getTicketById(String id) {
             return null;
         }
     }
+
+    public List<Ticket> getTicketsByStadiumSector(Ticket.StadiumSector stadiumSector) {
```
Comment (Alexmansar) [inline src/TicketService.java:52]: could be private

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@AnyaRatusnyak @Tolyaspit , Hi, please fix small comments

File: src/TicketService.java (hunk 2 above)
Comment (AnyaRatusnyak) [reply src/TicketService.java]: Thank you) Ok!

**Review by Alexmansar — APPROVED** [review]
@AnyaRatusnyak @Tolyaspit could be merged) Good job))

**Pull request merged**